openpype/hosts/max/plugins/load/load_redshift_proxy.py

```diff
@@ -0,0 +1,59 @@
+import os
+import clique
+
+from openpype.pipeline import (
+    load,
+    get_representation_path
+)
+from openpype.hosts.max.api.pipeline import containerise
+from openpype.hosts.max.api import lib
+
+
+class RedshiftProxyLoader(load.LoaderPlugin):
+    """Redshift Proxy Loader"""
+
+    families = ["redshiftproxy"]
+    representations = ["rs"]
+    order = -9
+    icon = "code-fork"
+    color = "white"
+
+    def load(self, context, name=None, namespace=None, data=None):
+        from pymxs import runtime as rt
+
+        filepath = os.path.normpath(self.fname)
```

Comment:
It would be good maybe to prepare for this PR #4602 and use `self.filepath_from_context(context)` instead of `self.fname`